shared/panels/project-update-sidepanel/project-update-sidepanel.en.json

```
@@ -43,5 +40,10 @@
         "codeReviews": "Code reviews"
       }
     }
+  },
+  "projectLeads": {
+    "title": "Project lead",
+    "me": "You",
```

Wouldn't this be simpler ? 😅 

```suggestion
    "you": "You",
```